ui/analyse/src/serverSideUnderboard.ts

```diff
@@ -69,6 +70,7 @@ export default function (element: HTMLElement, ctrl: AnalyseCtrl) {
           chart.lastPly = mainlinePly;
         }
       }
+      gameGifLink.pathname = `/game/export/gif/${ctrl.bottomColor()}/${data.game.id}.gif`;
```

Comment:
Seems to make more sense to move it further up where `positionGifLink` is set as well, inside the `if` which checks whether the fen actually changed.

Reply:
thank you, fixed